test/OpenTelemetry.Tests/Trace/BatchExportActivityProcessorTest.cs

```diff
@@ -190,7 +190,8 @@ public void CheckExportForRecordingButNotSampledActivity()
         [Fact]
         public void CheckExportDrainsBatchOnFailure()
         {
-            using var exporter = new InMemoryExporter<Activity>(null);
+            using var exporter = new InMemoryExporter<Activity>(exportFunc: x => ExportResult.Failure);
```

Comment:
Is it possible to just put `exportAction: null` (and avoid using Func)?

Reply:
I wouldn't use a null here, because I think then this test would be depending on side-effects to test the Failure behavior.
I like this one as it is because it clearly describes the behavior being tested here.
If you think this needs to be cleaned up, I could add an Extension method that just accepts the `ExportResult` enum.

Reply:
Or maybe we shouldn't use InMemoryExporter in this test case at all.

Reply:
I thought of that. :)
I thought it was a plus that this exporter was flexible enough to handle cases like this. 
But I have no problem putting a private custom exporter here.